### PR TITLE
Welcome tester emails

### PR DESCRIPTION
**Pull request by batandwa-bbd**
### This PR affects / introduces the following:
- Send notice emails to users who have been added as testers so they know how future notifications will come.

### CODE REVIEW

File: server/src/Repositories/UserProjectsRepository.cs
Comment (bbdnet2814) [inline server/src/Repositories/UserProjectsRepository.cs:1]: Take out the Role=3 part, the roles are 1 for tester and 2 for beta tester

File: server/src/Services/ReleaseTestersService.cs
Comment (bbdnet2814) [inline server/src/Services/ReleaseTestersService.cs:1]: Leave this file as it was as its just saving unnecessary calls the way it used to be

**Review by bbdnet2814 — DISMISSED** [review]
Look at comments

File: server/src/Repositories/UserProjectsRepository.cs
Comment (batandwa-bbd) [reply server/src/Repositories/UserProjectsRepository.cs:1]: 2 is already used for both

**Review by bbdnet2814 — APPROVED** [review]

**Pull request merged**